.dockerignore

```diff
@@ -0,0 +1 @@
+.git
```

Comment:
I learned that running `docker build` was pretty slow for this project, because [docker was sending the entire context/file tree from the host](https://stackoverflow.com/a/26604307), which included the entire `.git` folder. Adding this .dockerignore file sped it up tremendously!